modules/ROOT/pages/importword.adoc

```diff
@@ -0,0 +1,60 @@
+
+= Import from Word plugin
+:navtitle: Import from Word
+:description: Provides a way to import .docx (Word documents) or .dotx (Word templates) files into the editor.
+:description_short: Import .docx or .dotx files into the editor.
+:keywords: plugin, importword, import from word
+:pluginname: Import from Word
+:plugincode: importword
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+
+The {pluginname} plugin lets you import `.docx` (Word document) or `.dotx` (Word template) files into the editor. The process preserves formatting and rich media.
+
+== Interactive example
+
+[NOTE]
+.{pluginname} demo
+====
+Demos containing the {pluginname} plugin are currently only available on our main website. Check out https://www.tiny.cloud/tinymce/features/import-from-word/[the {pluginname} demo], or the https://www.tiny.cloud/[demo on our home page], to try it out today.
```

Comment:
```suggestion
Demos containing the {pluginname} plugin are currently only available on our main website. Check out https://www.tiny.cloud/tinymce/features/import-from-word/[the {pluginname} demo], to try it out today.
```

Reply:
@kemister85 Let me confirm if import from word is added to home page demo.

Reply:
As discussed, we will update this link in a seperate ticket, so we can commence QA for the rest of DOCs 7.0